packages/synapse-react-client/src/components/download_list/TableQueryDownloadConfirmation.test.tsx

```diff
@@ -265,9 +287,8 @@ describe('TableQueryDownloadConfirmation', () => {
       false,
     )
   })
-
-  it('setting the row primary key indicates that the row version should not be used', async () => {
-    await setUp('fileSynapseID')
```

Comment:
Test for this behavior is lost, but seems ok since it was added for PORTALS-2954, which was a workaround now resolved by using the feature under test